samples/net/sockets/http_get/sample.yaml

```diff
@@ -1,10 +1,21 @@
 sample:
   description: BSD Sockets API HTTP GET example
   name: socket_http_get
+common:
+  filter: TOOLCHAIN_HAS_NEWLIB == 1
+  harness: net
+  min_ram: 32
+  min_flash: 80
+  tags: net socket
 tests:
   sample.net.sockets.http_get:
+    harness: net
+  sample.net.sockets.http_get.posix:
     filter: TOOLCHAIN_HAS_NEWLIB == 1
     harness: net
```

Comment:
Most of the settings in `sample.net.sockets.http_get.posix` could be removed as they are now in common: block

Reply:
Ack, overlooked it, fixed.